libs/logging/src/logger.test.ts

```diff
@@ -396,3 +398,89 @@ describe('test cdf conversion', () => {
     `);
   });
 });
+
+describe('test cdf documentation generation', () => {
+  test('builds expected documentation for VxAdminApp', () => {
+    const logger = new Logger(LogSource.VxAdminApp);
+    const cdfDocumentationContent = logger.buildCDFLogDocumentationFileContent(
+      'machineID1234',
+      'VotingWorks',
+      'VxAdmin 1.0',
+      'codeversion'
+    );
+    const structuredDataResult = safeParseJson(
+      cdfDocumentationContent,
+      ElectionEventLogDocumentationSchema
+    );
+    expect(structuredDataResult.isOk()).toBeTruthy();
+    const structuredData = structuredDataResult.ok();
+    assert(structuredData);
```

Comment:
This will throw the error in the case that the result is not ok, whereas as written it will just assert `undefined`.
```suggestion
    const structuredData = structuredDataResult.unsafeUnwrap();
```